lighthouse-core/test/audits/load-fast-enough-for-pwa-test.js

```diff
@@ -51,7 +51,7 @@ describe('PWA: load-fast-enough-for-pwa audit', () => {
     return FastPWAAudit.audit(artifacts, {settings, computedCache: new Map()}).then(result => {
       assert.equal(result.score, false, 'not failing a long TTI value');
       assert.equal(result.rawValue, 15000);
-      assert.deepEqual(result.displayValue, ['Interactive at %d\xa0s', 15]);
+      expect(result.displayValue).toBeDisplayString('Interactive at 15.0\xa0s');
```

Comment:
let's add an assertion to `font-size-test` while we're at it so we'd catch the 4 digits thing